corehq/apps/domain/migrations/sql_templates/update_tables1.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS domain_registration_request CASCADE;
```

Comment:
I think that this shouldn't include CASCADE. CASCADE will drop anything that depends on this table, but right now we believe that nothing depends on this table

Reply:
Makes sense, removed `CASCADE` here: https://github.com/dimagi/commcare-hq/pull/20683/commits/b7ceac112610c9df648fe87393676634abddf97f